ci/run-k8s-e2e-tests.sh

```diff
@@ -36,6 +38,7 @@ using the sonobuoy tool.
         --e2e-focus TestRegex                                     Run only tests matching a specific regex, this is useful to run a single tests for example.
         --kubeconfig Kubeconfig                                   Explicit path to Kubeconfig file. You may also set the KUBECONFIG environment variable.
         --kube-conformance-image-version ConformanceImageVersion  Use specific version of the Conformance tests container image. Default is $KUBE_CONFORMANCE_IMAGE_VERSION.
+        --log-mode                                                Use the flag to set either report, detail, or dump level data for sonobouy results.
```

Comment:
Use quotes for the possible choices

'report', 'detail', 'dump'

Reply:
Done.